utils/spacewalk-common-channels.ini

```diff
@@ -3567,3 +3567,110 @@ gpgkey_url = http://download.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-8
 gpgkey_id = 2F86D6A1
 gpgkey_fingerprint = 94E2 79EB 8D8F 25B2 1810 ADF1 21EA 45AB 2F86 D6A1
 repo_url = http://mirrors.fedoraproject.org/mirrorlist?repo=epel-8&arch=%(arch)s
+
+[rockylinux9]
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = Rocky Linux 9 (%(arch)s)
+gpgkey_url = file:///etc/pki/rpm-gpg/RPM-GPG-KEY-Rocky-9
+gpgkey_id = 350D275D
+gpgkey_fingerprint = 21CB 256A E16F C54C 6E65  2949 702D 426D 350D 275D
+repo_url = https://mirrors.rockylinux.org/mirrorlist?repo=BaseOS-9&arch=%(arch)s
+dist_map_release = 9
+
+[rockylinux9-appstream]
+label    = %(base_channel)s-appstream
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = Rocky Linux 9 AppStream (%(arch)s)
+base_channels = rockylinux9-%(arch)s
+repo_url = https://mirrors.rockylinux.org/mirrorlist?repo=AppStream-9&arch=%(arch)s
+
+[rockylinux9-extras]
+label    = %(base_channel)s-extras
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = Rocky Linux 9 Extras (%(arch)s)
+base_channels = rockylinux9-%(arch)s
+repo_url = https://mirrors.rockylinux.org/mirrorlist?repo=extras-9&arch=%(arch)s
+
+[rockylinux9-crb]
+label    = %(base_channel)s-crb
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = Rocky Linux 9 CRB (%(arch)s)
+base_channels = rockylinux9-%(arch)s
+repo_url = https://mirrors.rockylinux.org/mirrorlist?repo=CRB-9&arch=%(arch)s
+
+[rockylinux9-highavailability]
+label    = %(base_channel)s-highavailability
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = Rocky Linux 9 High Availability (%(arch)s)
+base_channels = rockylinux9-%(arch)s
+repo_url = https://mirrors.rockylinux.org/mirrorlist?repo=HighAvailability-9&arch=%(arch)s
+
+[rockylinux9-nfv]
+label    = %(base_channel)s-nfv
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = Rocky Linux 9 NFV (%(arch)s)
+base_channels = rockylinux9-%(arch)s
+repo_url = https://mirrors.rockylinux.org/mirrorlist?repo=NFV-9&arch=%(arch)s
+
+[rockylinux9-plus]
+label    = %(base_channel)s-plus
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = Rocky Linux 9 Plus (%(arch)s)
+base_channels = rockylinux9-%(arch)s
+repo_url = https://mirrors.rockylinux.org/mirrorlist?repo=plus-9&arch=%(arch)s
+
+[rockylinux9-resilientstorage]
+label    = %(base_channel)s-resilientstorage
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = Rocky Linux 9 ResilientStorage (%(arch)s)
+base_channels = rockylinux9-%(arch)s
+repo_url = https://mirrors.rockylinux.org/mirrorlist?repo=ResilientStorage-9&arch=%(arch)sex
+
+[rockylinux9-rt]
+label    = %(base_channel)s-rt
+archs    = x86_64, aarch64, ppc64le, s390x
+name     = Rocky Linux 9 RT (%(arch)s)
+base_channels = rockylinux9-%(arch)s
+repo_url = https://mirrors.rockylinux.org/mirrorlist?repo=RT-9&arch=%(arch)s
+
+[rockylinux9-saphana]
```

Comment:
Does this actually exist? According to https://wiki.rockylinux.org/rocky/repo/, this is TBD, same for SAP and NFV

Reply:
Yes it does, I at the one part looked at the URIs in the actual files on a running system, but also checked them before adding 👍🏻